spec/system/reward_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reward check', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  it 'crud Reward for employee' do
+    visit root_path
+    click_link 'Log In'
```

Comment:
You are using login helpers in other specs - maybe use them here as well for the sake of consistenncy?

Reply:
I used the login helpers :)